src/components/image/index.tsx

```diff
@@ -196,8 +231,10 @@ class Image extends PureComponent<Props, State> {
           this.isGif() && styles.gifImage,
           aspectRatio && {aspectRatio},
           margins,
-          style
+          style,
+          {flexShrink}
```

Comment:
Why do we need to use flexShrink here? 
Btw, there's a big possibility that the asset you're working with in the public is not working exactly like the private one. 
For example I see the asset we have here only have 1 size instead of 5. 

Not sure it's related to the shrink think, but better copy the whole 5 sizes set.

Reply:
Added the whole 5.

`shrink` make the image live inside the boundaries of the wrapped container and keeping the same ratio.